# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1355007654 18000
#      Sat Dec 08 18:00:54 2012 -0500
# Node ID 534a2c881f45418f210e6ff6365c55d23c66345d
# Parent  ea1a1fb00744a8b73bb4856c1d93e6c04a720f4a
Increase test tolerance on sqp (bug #37742)

* scripts/optimization/sqp.m: Increase test tolerance.

diff --git a/scripts/optimization/sqp.m b/scripts/optimization/sqp.m
--- a/scripts/optimization/sqp.m
+++ b/scripts/optimization/sqp.m
@@ -743,17 +743,17 @@ endfunction
 %! x_opt = [-1.717143501952599;
 %!           1.595709610928535;
 %!           1.827245880097156;
 %!          -0.763643103133572;
 %!          -0.763643068453300];
 %!
 %! obj_opt = 0.0539498477702739;
 %!
-%! assert (x, x_opt, 5*sqrt (eps));
+%! assert (x, x_opt, 8*sqrt (eps));
 %! assert (obj, obj_opt, sqrt (eps));
 
 %% Test input validation
 %!error sqp ()
 %!error sqp (1)
 %!error sqp (1,2,3,4,5,6,7,8,9)
 %!error sqp (1,2,3,4,5)
 %!error sqp (ones (2,2))
